unstructured/partition/pdf.py

```diff
@@ -172,16 +172,16 @@ def partition_pdf(
         with Tesseract, you'll first need to install the appropriate Tesseract language pack.
     metadata_last_modified
         The last modified date for the document.
-    extract_images_in_pdf
```

Comment:
let's keep the old kwarg around as a deprecated argument for awhile to avoid breaking changes.